distribution/changelog.txt

```diff
@@ -1,6 +1,7 @@
 0.4.3 (in development)
 ------------------------------------------------------------------------
 - Feature: [#17782] The Flying Coaster now has access to boosters and can draw outside loops.
+- Feature: [#18013,18016,18018,18019,18514] Added colour presets to Spiral Slide, Dodgems, Boat Hire, Flying Saucers, and Car Ride.
```

Comment:
```suggestion
- Improved: [#18013, #18016, #18018, #18019, #18514] Added colour presets to Spiral Slide, Dodgems, Boat Hire, Flying Saucers, and Car Ride.
```

Note that you will have to move it down to be in order, and that you have to rebase or use the GitHub-provided option to resolve conflicts.

Reply:
I have updated it